src/hangman/game.clj

```diff
@@ -8,26 +8,28 @@
 
 (def game-id (atom nil))
 
+(defn valid-user-input? [user-input]
+  (let [valid? (= (count user-input) 1)]
+    (when (not valid?) (println "Invalid input"))
+    valid?))
+
+(defn valid-chars [] (available-letters @game-id))
+
+(defn valid-character? [character]
+  (let [valid? (contains? (valid-chars) character)]
```

Comment:
Same thing here, in plus these two functions could probably be refactored in one, I can do it after if you don't see how..